connection.go

```diff
@@ -281,14 +282,68 @@ func (c *connection) close(ctx context.Context) error {
 func (c *connection) login(ctx context.Context) error {
 	hasCompression := c.config.compression
 	c.config.compression = false
+
+	authRequest := AuthCommand{
+		UseCompression: false,
+		ClientName:     c.config.clientName,
+		DriverName:     fmt.Sprintf("exasol-driver-go %s", driverVersion),
+		ClientOs:       runtime.GOOS,
+		ClientVersion:  "(unknown version)",
+		ClientRuntime:  runtime.Version(),
+		Attributes: Attributes{
+			Autocommit:         boolToPtr(c.config.autocommit),
+			CurrentSchema:      c.config.schema,
+			CompressionEnabled: boolToPtr(hasCompression),
+		},
+	}
+
+	if c.config.accessToken != "" {
```

Comment:
Is it possible to extract this `if` into a separate method that returns the auth request and an error?